checker/build.xml

```diff
@@ -550,6 +550,8 @@
               </fileset>
           </batchtest>
       </junit>
+
+      <antcall target="units_extension-tests"/>
```

Comment:
Tainting and Regex have underscores in their targets:
regex-tests
regex_classic-tests
regex_qual-tests
tainting-tests
tainting_classic-tests
tainting_qual-tests

Should I fix those up as well?